Source/Blazorise/Components/MessageAlert/MessageAlert.razor.cs

```diff
@@ -100,6 +102,26 @@ protected Task OnCancelClicked()
         } );
     }
 
+    /// <summary>
+    /// Handles the event when a choice button is clicked, hiding the modal and invoking callbacks as necessary.
+    /// </summary>
+    /// <param name="choice">Represents the button that was clicked, providing its key for further processing.</param>
+    /// <returns>Returns a task that represents the asynchronous operation of handling the button click.</returns>
+    protected Task OnChoiceClicked( MessageOptionsChoice choice )
+    {
+        return InvokeAsync( async () =>
+        {
+            await ModalRef.Hide();
+
+            if ( IsChoice && Callback is not null && !Callback.Task.IsCompleted )
```

Comment:
I wonder if there is a case there the `Callback.Task.IsCompleted` would be null?

Reply:
I've checked the internals of `TaskCompletionSource`, and Task is `readonly` and initialized in the constructor. I doubt it can ever be null.